### DISCUSSION TITLE
The front end request returns 200, and the log prompt times out

### DISCUSSION THREAD

**Discussion by cj12580** [apache/skywalking#8279] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

NodeJS Client Side Agent (apache/skywalking-client-js)

### What happened

![image](https://user-images.githubusercontent.com/19181652/145509759-2bde31ab-226c-43a4-9d14-cb3749e11892.png)

![image](https://user-images.githubusercontent.com/19181652/145509866-8ecdfd70-aca5-4c07-b5a5-b6a366b0ecf1.png)

The content cannot be reported during front-end Vue integration. The returned code is 200, but the log prompt timeout

### What you expected to happen

There may be a problem with the reported parameters

### How to reproduce

1. Front end Vue project configuration completed

2. Click a button to trigger error log reporting

3. Trigger / Browser / errorlogs interface

4. 200 is returned, but the reported data is not viewed in the background. The log view prompt times out

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
If the browser says OK, then the data reported. Did you check whether there is gateway or fireway something to affect? And do you have any check on OAP logs, and health check.

**Comment by cj12580** [comment]
Hello, can this trace monitor the front-end link?





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/skywalking"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月10日(星期五) 中午11:26
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/skywalking] The front end request returns 200, and the log prompt times out (Discussion #8279)





 
If the browser says OK, then the data reported. Did you check whether there is gateway or fireway something to affect? And do you have any check on OAP logs, and health check.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by wu-sheng** [reply]
If you want to try something, demo env and quick start(local k8s) are both provided. Try from there.
